src/utils/formik-helpers/formik-helpers.ts

```diff
@@ -185,7 +203,9 @@ const touchedResourceTabFields = (registration: Registration): FormikTouched<unk
           },
         },
       };
-    case PublicationType.Degree:
+    case PublicationType.Degree: {
+      const publicationInstance = registration.entityDescription?.reference
```

Comment:
greit å gi eksplisitt navn, så man ikke ser feil når `publicationInstance` nevnes så mange ganger nedover her
```suggestion
      const degreePublicationInstance = registration.entityDescription?.reference
```